src/db/migrations/20210307130019-create-article.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
Can just delete this file because it'll be superseded by the next migration anyway.